src/command_link.rs

```diff
@@ -27,24 +27,56 @@ pub fn run_command_link(
     }
 
-    let absolute_file_path = Path::new(file)
-        .absolutize()
-        .with_context(|| format!("Failed to convert path `{}` to absolute path.", file))?;
+    // Check if this is a channel alias (starts with +)
+    if let Some(target_channel) = target.strip_prefix('+') {
+        // Validate that the target channel exists and is not an alias
+        if let Some(target_info) = config_file.data.installed_channels.get(target_channel) {
+            // Prevent alias-to-alias chains for simplicity and maintainability
+            if let JuliaupConfigChannel::AliasChannel { .. } = target_info {
+                bail!("Cannot create an alias to another alias `{}`. Please create an alias directly to the target channel instead.", target_channel);
+            }
+        } else if !is_valid_channel(&versiondb_data, &target_channel.to_string())? {
+            bail!("Target channel `{}` is not installed and is not a valid system channel. Please run `juliaup add {}` first or check `juliaup list` for available channels.", target_channel, target_channel);
+        }
 
-    if !is_valid_julia_path(&absolute_file_path.to_path_buf()) {
-        eprintln!("WARNING: There is no julia binary at {}. If this was a mistake, run `juliaup remove {}` and try again.", absolute_file_path.to_string_lossy(), channel);
-    }
+        if !args.is_empty() {
+            bail!("Arguments are not supported when creating channel aliases. Remove the extra arguments: {:?}", args);
+        }
+
+        config_file.data.installed_channels.insert(
+            channel.to_string(),
+            JuliaupConfigChannel::AliasChannel {
+                target: target_channel.to_string(),
+            },
+        );
+
+        eprintln!("Channel alias `{channel}` created, pointing to `{target_channel}`.");
+    } else {
+        let absolute_file_path = Path::new(target)
+            .absolutize()
+            .with_context(|| format!("Failed to convert path `{target}` to absolute path."))?;
+
+        if !is_valid_julia_path(&absolute_file_path.to_path_buf()) {
+            eprintln!("WARNING: There is no julia binary at {}. If this was a mistake, run `juliaup remove {}` and try again.", absolute_file_path.to_string_lossy(), channel);
+        }
 
-    config_file.data.installed_channels.insert(
-        channel.to_string(),
-        JuliaupConfigChannel::LinkedChannel {
-            command: absolute_file_path.to_string_lossy().to_string(),
-            args: Some(args.to_vec()),
-        },
-    );
+        config_file.data.installed_channels.insert(
+            channel.to_string(),
+            JuliaupConfigChannel::LinkedChannel {
+                command: absolute_file_path.to_string_lossy().to_string(),
+                args: Some(args.to_vec()),
+            },
+        );
+
+        eprintln!(
+            "Channel `{}` linked to `{}`.",
```

Comment:
These keep switching back and forth in convention. Maybe we just do a standardized convention later?

Reply:
Maybe a .pre-commit-config.yaml file? I don't know AGENTS.md